### PR TITLE
update params structure for OSM hosted iD

### LINKED ISSUE #9093: [Solved] gpx parameter in URL causes Application error
### URL

https://www.openstreetmap.org/edit?gpx=https%3A//server.nikhilvj.co.in/pmgsy/gpx/1445.gpx#map=16/28.1952/80.6254

### How to reproduce the issue?

1. Following the documentation at https://github.com/openstreetmap/iD/blob/develop/API.md
2. Generating a URL of iD Editor with a working .gpx file's URL included in URL Params
3. Getting error page:
```
Application error
The OpenStreetMap server encountered an unexpected condition that prevented it from fulfilling the request (HTTP 500)

Feel free to [contact](http://wiki.openstreetmap.org/wiki/Contact) the OpenStreetMap community if your problem persists. Make a note of the exact URL / post data of your request.

This may be a problem in our Ruby On Rails code. 500 occurs with exceptions thrown outside of an action (like in Dispatcher setups or broken Ruby code)
```

GPX file URL: https://server.nikhilvj.co.in/pmgsy/gpx/1445.gpx  
Loaded it up on https://geojson.io and confirmed that it renders properly over there -> assuming it's a valid GPX then. (please tell if not, and what format is reqd)  

Logged in to Openstreetmap already? : Yes  

Following URL param configurations fail:
- Unaltered gpx URL:
- - https://www.openstreetmap.org/edit?map=16/28.1952/80.6254&gpx=https://server.nikhilvj.co.in/pmgsy/gpx/1445.gpx
- - https://www.openstreetmap.org/edit?gpx=https://server.nikhilvj.co.in/pmgsy/gpx/1445.gpx

- encoded GPX URL: (just the : is replaced by %3A)
- - https://www.openstreetmap.org/edit?map=16/28.1952/80.6254&gpx=https%3A//server.nikhilvj.co.in/pmgsy/gpx/1445.gpx
- - https://www.openstreetmap.org/edit?gpx=https%3A//server.nikhilvj.co.in/pmgsy/gpx/1445.gpx

### Screenshot(s) or anything else?

Screenshot_2022-05-01_17-14-43

Contents of GPX file:  
<details>

```
<?xml version="1.0"?>
<gpx version="1.1" creator="GDAL 3.4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.topografix.com/GPX/1/1" xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd">
<metadata><bounds minlat="18.601548052999998" minlon="81.648038112999998" maxlat="19.010908051000001" maxlon="82.144620493999994"/></metadata>                  
<trk>
  <trkseg>
    <trkpt lat="19.002683533" lon="81.976895614">
    </trkpt>
    <trkpt lat="18.996481867" lon="81.973134297">
    </trkpt>
    <trkpt lat="18.986310677" lon="81.971660611">
    </trkpt>
    <trkpt lat="18.98547297" lon="81.972683377">
    </trkpt>
    <trkpt lat="18.977861786" lon="81.968920493">
    </trkpt>
    <trkpt lat="18.978471914" lon="81.966514573">
    </trkpt>
    <trkpt lat="18.975456281" lon="81.964065497">
    </trkpt>
    <trkpt lat="18.968044687" lon="81.961433219">
    </trkpt>
    <trkpt lat="18.958413299" lon="81.962061529">
    </trkpt>
    <trkpt lat="18.951614849" lon="81.955541013">
    </trkpt>
    <trkpt lat="18.947827556" lon="81.947650041">
    </trkpt>
    <trkpt lat="18.944957588" lon="81.945129048">
    </trkpt>
    <trkpt lat="18.930890208" lon="81.946087936">
    </trkpt>
    <trkpt lat="18.921794394" lon="81.951224198">
    </trkpt>
    <trkpt lat="18.920286393" lon="81.953743306">
    </trkpt>
    <trkpt lat="18.926504526" lon="81.95482718">
    </trkpt>
    <trkpt lat="18.935950471" lon="81.966649757">
    </trkpt>
    <trkpt lat="18.934270009" lon="81.969340436">
    </trkpt>
    <trkpt lat="18.928173731" lon="81.9687094">
    </trkpt>
    <trkpt lat="18.925424373" lon="81.969958215">
    </trkpt>
    <trkpt lat="18.915977355" lon="81.977213622">
    </trkpt>
    <trkpt lat="18.916277215" lon="81.980789374">
    </trkpt>
    <trkpt lat="18.919913858" lon="81.982165506">
    </trkpt>
    <trkpt lat="18.923428059" lon="81.986179623">
    </trkpt>
    <trkpt lat="18.9231234" lon="82.001115783">
    </trkpt>
    <trkpt lat="18.926648718" lon="82.002960372">
    </trkpt>
    <trkpt lat="18.912910767" lon="82.018359828">
    </trkpt>
    <trkpt lat="18.913993964" lon="82.020675041">
    </trkpt>
    <trkpt lat="18.910709863" lon="82.023993976">
    </trkpt>
    <trkpt lat="18.912595092" lon="82.032305377">
    </trkpt>
    <trkpt lat="18.911801567" lon="82.034973175">
    </trkpt>
    <trkpt lat="18.907670595" lon="82.038518494">
    </trkpt>
    <trkpt lat="18.902680984" lon="82.039160424">
    </trkpt>
    <trkpt lat="18.896675599" lon="82.038702011">
    </trkpt>
    <trkpt lat="18.89294892" lon="82.029005329">
    </trkpt>
    <trkpt lat="18.893443646" lon="82.025000956">
    </trkpt>
    <trkpt lat="18.898989897" lon="82.01697258">
    </trkpt>
    <trkpt lat="18.898151434" lon="82.009983654">
    </trkpt>
    <trkpt lat="18.892947809" lon="82.00565356">
    </trkpt>
    <trkpt lat="18.886036894" lon="82.007326793">
    </trkpt>
    <trkpt lat="18.877201329" lon="82.006425081">
    </trkpt>
    <trkpt lat="18.874468735" lon="81.995265125">
    </trkpt>
    <trkpt lat="18.87507693" lon="81.990049121">
    </trkpt>
    <trkpt lat="18.880181462" lon="81.985592698">
    </trkpt>
    <trkpt lat="18.881052728" lon="81.979161876">
    </trkpt>
    <trkpt lat="18.875019359" lon="81.979944304">
    </trkpt>
    <trkpt lat="18.864788503" lon="81.971115408">
    </trkpt>
    <trkpt lat="18.861172321" lon="81.980476608">
    </trkpt>
    <trkpt lat="18.839807507" lon="81.985588099">
    </trkpt>
    <trkpt lat="18.836895936" lon="81.984579755">
    </trkpt>
    <trkpt lat="18.836042228" lon="81.986226029">
    </trkpt>
    <trkpt lat="18.830987246" lon="81.984005837">
    </trkpt>
    <trkpt lat="18.826590148" lon="81.984441958">
    </trkpt>
    <trkpt lat="18.825665511" lon="81.987072156">
    </trkpt>
    <trkpt lat="18.822176696" lon="81.986740264">
    </trkpt>
    <trkpt lat="18.818783896" lon="81.99536199">
    </trkpt>
    <trkpt lat="18.825592331" lon="82.028651467">
    </trkpt>
    <trkpt lat="18.821142098" lon="82.03791559">
    </trkpt>
    <trkpt lat="18.815118466" lon="82.045807642">
    </trkpt>
    <trkpt lat="18.819086657" lon="82.051708555">
    </trkpt>
    <trkpt lat="18.815770281" lon="82.060032584">
    </trkpt>
    <trkpt lat="18.821293795" lon="82.100116717">
    </trkpt>
    <trkpt lat="18.812764274" lon="82.096689152">
    </trkpt>
    <trkpt lat="18.801942737" lon="82.095314286">
    </trkpt>
    <trkpt lat="18.795832038" lon="82.103369243">
    </trkpt>
    <trkpt lat="18.810242704" lon="82.11557159">
    </trkpt>
    <trkpt lat="18.819747415" lon="82.113381367">
    </trkpt>
    <trkpt lat="18.817538717" lon="82.120216729">
    </trkpt>
    <trkpt lat="18.808726551" lon="82.132593399">
    </trkpt>
    <trkpt lat="18.78916492" lon="82.142909095">
    </trkpt>
    <trkpt lat="18.780190628" lon="82.144620494">
    </trkpt>
    <trkpt lat="18.77838316" lon="82.1387051">
    </trkpt>
    <trkpt lat="18.781527231" lon="82.136211179">
    </trkpt>
    <trkpt lat="18.780550312" lon="82.133346084">
    </trkpt>
    <trkpt lat="18.776251327" lon="82.133585196">
    </trkpt>
    <trkpt lat="18.770549837" lon="82.131390313">
    </trkpt>
    <trkpt lat="18.768301195" lon="82.133524883">
    </trkpt>
    <trkpt lat="18.765810146" lon="82.133242193">
    </trkpt>
    <trkpt lat="18.770077261" lon="82.119967384">
    </trkpt>
    <trkpt lat="18.767960615" lon="82.112003731">
    </trkpt>
    <trkpt lat="18.765971992" lon="82.110703566">
    </trkpt>
    <trkpt lat="18.767557114" lon="82.105877807">
    </trkpt>
    <trkpt lat="18.764028364" lon="82.098012185">
    </trkpt>
    <trkpt lat="18.766189956" lon="82.093072126">
    </trkpt>
    <trkpt lat="18.765100965" lon="82.090795622">
    </trkpt>
    <trkpt lat="18.760756809" lon="82.088524804">
    </trkpt>
    <trkpt lat="18.75208318" lon="82.092336703">
    </trkpt>
    <trkpt lat="18.741957541" lon="82.093113254">
    </trkpt>
    <trkpt lat="18.726393604" lon="82.086306361">
    </trkpt>
    <trkpt lat="18.72205553" lon="82.087452769">
    </trkpt>
    <trkpt lat="18.719159483" lon="82.085939354">
    </trkpt>
    <trkpt lat="18.725999456" lon="82.068464643">
    </trkpt>
    <trkpt lat="18.71948079" lon="82.063921447">
    </trkpt>
    <trkpt lat="18.718025394" lon="82.059368958">
    </trkpt>
    <trkpt lat="18.722716785" lon="82.054045346">
    </trkpt>
    <trkpt lat="18.721258855" lon="82.048354023">
    </trkpt>
    <trkpt lat="18.723410476" lon="82.039238997">
    </trkpt>
    <trkpt lat="18.723026717" lon="82.028610624">
    </trkpt>
    <trkpt lat="18.721565486" lon="82.021780758">
    </trkpt>
    <trkpt lat="18.713218914" lon="82.008892857">
    </trkpt>
    <trkpt lat="18.713565222" lon="82.002059522">
    </trkpt>
    <trkpt lat="18.70450624" lon="81.994489">
    </trkpt>
    <trkpt lat="18.705213634" lon="81.987655206">
    </trkpt>
    <trkpt lat="18.696650327" lon="81.967617451">
    </trkpt>
    <trkpt lat="18.709790612" lon="81.958389145">
    </trkpt>
    <trkpt lat="18.719534155" lon="81.94233471">
    </trkpt>
    <trkpt lat="18.7269818" lon="81.943959377">
    </trkpt>
    <trkpt lat="18.730224659" lon="81.941120329">
    </trkpt>
    <trkpt lat="18.733106084" lon="81.941228729">
    </trkpt>
    <trkpt lat="18.753844218" lon="81.956900046">
    </trkpt>
    <trkpt lat="18.761621966" lon="81.954965769">
    </trkpt>
    <trkpt lat="18.765888518" lon="81.949515397">
    </trkpt>
    <trkpt lat="18.7694001" lon="81.948805639">
    </trkpt>
    <trkpt lat="18.771239067" lon="81.946139133">
    </trkpt>
    <trkpt lat="18.773756998" lon="81.935556786">
    </trkpt>
    <trkpt lat="18.774783014" lon="81.920308611">
    </trkpt>
    <trkpt lat="18.759723028" lon="81.910242846">
    </trkpt>
    <trkpt lat="18.758481521" lon="81.898647802">
    </trkpt>
    <trkpt lat="18.754244633" lon="81.891892841">
    </trkpt>
    <trkpt lat="18.757781103" lon="81.879063731">
    </trkpt>
    <trkpt lat="18.763862885" lon="81.872115627">
    </trkpt>
    <trkpt lat="18.759203654" lon="81.869742557">
    </trkpt>
    <trkpt lat="18.747617201" lon="81.868047744">
    </trkpt>
    <trkpt lat="18.744286775" lon="81.864364795">
    </trkpt>
    <trkpt lat="18.738038144" lon="81.849580338">
    </trkpt>
    <trkpt lat="18.740073104" lon="81.834016839">
    </trkpt>
    <trkpt lat="18.745156426" lon="81.820930239">
    </trkpt>
    <trkpt lat="18.739636269" lon="81.809408682">
    </trkpt>
    <trkpt lat="18.738862113" lon="81.802474106">
    </trkpt>
    <trkpt lat="18.744628727" lon="81.799179835">
    </trkpt>
    <trkpt lat="18.75118827" lon="81.790352343">
    </trkpt>
    <trkpt lat="18.756865804" lon="81.771768665">
    </trkpt>
    <trkpt lat="18.764154199" lon="81.776161501">
    </trkpt>
    <trkpt lat="18.772756235" lon="81.776807383">
    </trkpt>
    <trkpt lat="18.769515095" lon="81.770746678">
    </trkpt>
    <trkpt lat="18.77274086" lon="81.759794758">
    </trkpt>
    <trkpt lat="18.785089205" lon="81.766599876">
    </trkpt>
    <trkpt lat="18.795089676" lon="81.761270452">
    </trkpt>
    <trkpt lat="18.798286569" lon="81.754824603">
    </trkpt>
    <trkpt lat="18.8091328" lon="81.757277928">
    </trkpt>
    <trkpt lat="18.816649839" lon="81.746427175">
    </trkpt>
    <trkpt lat="18.818250309" lon="81.740272812">
    </trkpt>
    <trkpt lat="18.821887796" lon="81.738196464">
    </trkpt>
    <trkpt lat="18.826875452" lon="81.728703517">
    </trkpt>
    <trkpt lat="18.827434489" lon="81.721849217">
    </trkpt>
    <trkpt lat="18.833616665" lon="81.719693042">
    </trkpt>
    <trkpt lat="18.835617617" lon="81.71737734">
    </trkpt>
    <trkpt lat="18.838143368" lon="81.71206536">
    </trkpt>
    <trkpt lat="18.833040434" lon="81.709976086">
    </trkpt>
    <trkpt lat="18.825876871" lon="81.703308135">
    </trkpt>
    <trkpt lat="18.824618404" lon="81.703929494">
    </trkpt>
    <trkpt lat="18.820909599" lon="81.696572895">
    </trkpt>
    <trkpt lat="18.823087152" lon="81.687501897">
    </trkpt>
    <trkpt lat="18.819797171" lon="81.683226579">
    </trkpt>
    <trkpt lat="18.823884061" lon="81.678217119">
    </trkpt>
    <trkpt lat="18.823390855" lon="81.670164349">
    </trkpt>
    <trkpt lat="18.827555702" lon="81.660549857">
    </trkpt>
    <trkpt lat="18.829322877" lon="81.660031354">
    </trkpt>
    <trkpt lat="18.833459181" lon="81.662785293">
    </trkpt>
    <trkpt lat="18.838304139" lon="81.674045385">
    </trkpt>
    <trkpt lat="18.841215366" lon="81.674112429">
    </trkpt>
    <trkpt lat="18.840809508" lon="81.679983311">
    </trkpt>
    <trkpt lat="18.842378769" lon="81.683266532">
    </trkpt>
    <trkpt lat="18.849008316" lon="81.688065865">
    </trkpt>
    <trkpt lat="18.850809031" lon="81.683195013">
    </trkpt>
    <trkpt lat="18.860191" lon="81.672699612">
    </trkpt>
    <trkpt lat="18.866142419" lon="81.657791201">
    </trkpt>
    <trkpt lat="18.872720791" lon="81.662190001">
    </trkpt>
    <trkpt lat="18.874286597" lon="81.661992903">
    </trkpt>
    <trkpt lat="18.878142568" lon="81.655655835">
    </trkpt>
    <trkpt lat="18.885245994" lon="81.66119476">
    </trkpt>
    <trkpt lat="18.887579677" lon="81.660039002">
    </trkpt>
    <trkpt lat="18.890239342" lon="81.664007865">
    </trkpt>
    <trkpt lat="18.895965462" lon="81.666401858">
    </trkpt>
    <trkpt lat="18.892490258" lon="81.675660981">
    </trkpt>
    <trkpt lat="18.895295134" lon="81.679851345">
    </trkpt>
    <trkpt lat="18.911316063" lon="81.686984317">
    </trkpt>
    <trkpt lat="18.910154165" lon="81.693730588">
    </trkpt>
    <trkpt lat="18.9159939" lon="81.700031284">
    </trkpt>
    <trkpt lat="18.924657247" lon="81.704305122">
    </trkpt>
    <trkpt lat="18.923418776" lon="81.70725713">
    </trkpt>
    <trkpt lat="18.924143108" lon="81.711244653">
    </trkpt>
    <trkpt lat="18.935856467" lon="81.717534767">
    </trkpt>
    <trkpt lat="18.937084204" lon="81.721161542">
    </trkpt>
    <trkpt lat="18.936421206" lon="81.727049413">
    </trkpt>
    <trkpt lat="18.940220223" lon="81.72846711">
    </trkpt>
    <trkpt lat="18.954280313" lon="81.729292877">
    </trkpt>
    <trkpt lat="18.961908134" lon="81.727769798">
    </trkpt>
    <trkpt lat="18.970920715" lon="81.732837015">
    </trkpt>
    <trkpt lat="18.978940878" lon="81.732314382">
    </trkpt>
    <trkpt lat="18.980382516" lon="81.730152179">
    </trkpt>
    <trkpt lat="18.980227417" lon="81.723348523">
    </trkpt>
    <trkpt lat="18.975836422" lon="81.721716963">
    </trkpt>
    <trkpt lat="18.978816446" lon="81.718410298">
    </trkpt>
    <trkpt lat="18.980991545" lon="81.706910656">
    </trkpt>
    <trkpt lat="18.986392642" lon="81.701028232">
    </trkpt>
    <trkpt lat="18.989746821" lon="81.699701209">
    </trkpt>
    <trkpt lat="18.997787558" lon="81.705791761">
    </trkpt>
    <trkpt lat="18.994209419" lon="81.71924191">
    </trkpt>
    <trkpt lat="18.997460925" lon="81.723028076">
    </trkpt>
    <trkpt lat="19.005573373" lon="81.723538092">
    </trkpt>
    <trkpt lat="19.00745949" lon="81.725320526">
    </trkpt>
    <trkpt lat="19.00322613" lon="81.736060223">
    </trkpt>
    <trkpt lat="18.999639895" lon="81.757649148">
    </trkpt>
    <trkpt lat="18.987666727" lon="81.769664887">
    </trkpt>
    <trkpt lat="18.986723938" lon="81.772299426">
    </trkpt>
    <trkpt lat="18.986388205" lon="81.775877758">
    </trkpt>
    <trkpt lat="18.992031493" lon="81.776928507">
    </trkpt>
    <trkpt lat="18.993436921" lon="81.789226533">
    </trkpt>
    <trkpt lat="18.990674507" lon="81.794033822">
    </trkpt>
    <trkpt lat="18.985673372" lon="81.793599309">
    </trkpt>
    <trkpt lat="18.982220002" lon="81.795415937">
    </trkpt>
    <trkpt lat="18.973105595" lon="81.81488403">
    </trkpt>
    <trkpt lat="18.971382919" lon="81.822323393">
    </trkpt>
    <trkpt lat="18.973524007" lon="81.826691565">
    </trkpt>
    <trkpt lat="18.982860688" lon="81.826193018">
    </trkpt>
    <trkpt lat="18.98328625" lon="81.830342503">
    </trkpt>
    <trkpt lat="18.991627124" lon="81.84022049">
    </trkpt>
    <trkpt lat="18.984235249" lon="81.859293018">
    </trkpt>
    <trkpt lat="18.97490341" lon="81.863045473">
    </trkpt>
    <trkpt lat="18.96947115" lon="81.862706205">
    </trkpt>
    <trkpt lat="18.964820562" lon="81.872235081">
    </trkpt>
    <trkpt lat="18.965373409" lon="81.880225657">
    </trkpt>
    <trkpt lat="18.963446498" lon="81.880062064">
    </trkpt>
    <trkpt lat="18.962208169" lon="81.884963663">
    </trkpt>
    <trkpt lat="18.964190228" lon="81.892386435">
    </trkpt>
    <trkpt lat="18.966958434" lon="81.892110031">
    </trkpt>
    <trkpt lat="18.970539907" lon="81.89525009">
    </trkpt>
    <trkpt lat="18.980294721" lon="81.905331491">
    </trkpt>
    <trkpt lat="18.98001292" lon="81.908516674">
    </trkpt>
    <trkpt lat="18.98225642" lon="81.906817678">
    </trkpt>
    <trkpt lat="18.993646404" lon="81.907873049">
    </trkpt>
    <trkpt lat="18.991970925" lon="81.919036369">
    </trkpt>
    <trkpt lat="19.00654762" lon="81.919524481">
    </trkpt>
    <trkpt lat="19.006879794" lon="81.92425047">
    </trkpt>
    <trkpt lat="19.010908051" lon="81.926100656">
    </trkpt>
    <trkpt lat="19.008433421" lon="81.936883033">
    </trkpt>
    <trkpt lat="19.00639614" lon="81.935901399">
    </trkpt>
    <trkpt lat="18.99787147" lon="81.941220908">
    </trkpt>
    <trkpt lat="18.999135273" lon="81.945117414">
    </trkpt>
    <trkpt lat="19.005557182" lon="81.947017584">
    </trkpt>
    <trkpt lat="19.001546683" lon="81.96348481">
    </trkpt>
    <trkpt lat="19.002683533" lon="81.976895614">
    </trkpt>
  </trkseg>
  <trkseg>
    <trkpt lat="18.661242568" lon="81.900639647">
    </trkpt>
    <trkpt lat="18.658097309" lon="81.898086293">
    </trkpt>
    <trkpt lat="18.657454542" lon="81.893948167">
    </trkpt>
    <trkpt lat="18.655257554" lon="81.893199261">
    </trkpt>
    <trkpt lat="18.650764671" lon="81.87664113">
    </trkpt>
    <trkpt lat="18.643985523" lon="81.869852777">
    </trkpt>
    <trkpt lat="18.633281401" lon="81.847493842">
    </trkpt>
    <trkpt lat="18.626719603" lon="81.840229733">
    </trkpt>
    <trkpt lat="18.626415565" lon="81.829640696">
    </trkpt>
    <trkpt lat="18.622410383" lon="81.823854244">
    </trkpt>
    <trkpt lat="18.618541794" lon="81.813198764">
    </trkpt>
    <trkpt lat="18.618576894" lon="81.783997111">
    </trkpt>
    <trkpt lat="18.614359061" lon="81.775174147">
    </trkpt>
    <trkpt lat="18.61398085" lon="81.765904137">
    </trkpt>
    <trkpt lat="18.609487211" lon="81.753681073">
    </trkpt>
    <trkpt lat="18.607106326" lon="81.734435564">
    </trkpt>
    <trkpt lat="18.61165297" lon="81.705164356">
    </trkpt>
    <trkpt lat="18.605193554" lon="81.695682265">
    </trkpt>
    <trkpt lat="18.604417817" lon="81.687891256">
    </trkpt>
    <trkpt lat="18.601548053" lon="81.680458963">
    </trkpt>
    <trkpt lat="18.60831586" lon="81.674845555">
    </trkpt>
    <trkpt lat="18.609739687" lon="81.669270529">
    </trkpt>
    <trkpt lat="18.608323526" lon="81.661705039">
    </trkpt>
    <trkpt lat="18.614175684" lon="81.64931118">
    </trkpt>
    <trkpt lat="18.627333793" lon="81.648038113">
    </trkpt>
    <trkpt lat="18.632112445" lon="81.649837181">
    </trkpt>
    <trkpt lat="18.634136703" lon="81.654479103">
    </trkpt>
    <trkpt lat="18.636337677" lon="81.655871405">
    </trkpt>
    <trkpt lat="18.646499895" lon="81.655980464">
    </trkpt>
    <trkpt lat="18.656486676" lon="81.659194888">
    </trkpt>
    <trkpt lat="18.651771122" lon="81.666061162">
    </trkpt>
    <trkpt lat="18.649949294" lon="81.673249472">
    </trkpt>
    <trkpt lat="18.653008007" lon="81.675699851">
    </trkpt>
    <trkpt lat="18.658627963" lon="81.674527397">
    </trkpt>
    <trkpt lat="18.661050475" lon="81.678673836">
    </trkpt>
    <trkpt lat="18.664978717" lon="81.680951492">
    </trkpt>
    <trkpt lat="18.665721932" lon="81.687130997">
    </trkpt>
    <trkpt lat="18.670318133" lon="81.69188935">
    </trkpt>
    <trkpt lat="18.675687737" lon="81.704210643">
    </trkpt>
    <trkpt lat="18.687346727" lon="81.708971057">
    </trkpt>
    <trkpt lat="18.690145416" lon="81.711629564">
    </trkpt>
    <trkpt lat="18.688905576" lon="81.71492893">
    </trkpt>
    <trkpt lat="18.666241" lon="81.73181465">
    </trkpt>
    <trkpt lat="18.663176837" lon="81.740096244">
    </trkpt>
    <trkpt lat="18.657082389" lon="81.746434651">
    </trkpt>
    <trkpt lat="18.652713143" lon="81.757085093">
    </trkpt>
    <trkpt lat="18.646701943" lon="81.798063573">
    </trkpt>
    <trkpt lat="18.643142271" lon="81.804318884">
    </trkpt>
    <trkpt lat="18.652969055" lon="81.811613373">
    </trkpt>
    <trkpt lat="18.664517125" lon="81.823775476">
    </trkpt>
    <trkpt lat="18.675217409" lon="81.827980337">
    </trkpt>
    <trkpt lat="18.679785631" lon="81.832980276">
    </trkpt>
    <trkpt lat="18.681778253" lon="81.84060341">
    </trkpt>
    <trkpt lat="18.687570267" lon="81.850575229">
    </trkpt>
    <trkpt lat="18.694606233" lon="81.853705159">
    </trkpt>
    <trkpt lat="18.696988821" lon="81.862780129">
    </trkpt>
    <trkpt lat="18.706608297" lon="81.86816406">
    </trkpt>
    <trkpt lat="18.72491476" lon="81.871517937">
    </trkpt>
    <trkpt lat="18.723166116" lon="81.877050735">
    </trkpt>
    <trkpt lat="18.728423646" lon="81.878108179">
    </trkpt>
    <trkpt lat="18.721788791" lon="81.889428996">
    </trkpt>
    <trkpt lat="18.718172917" lon="81.901355608">
    </trkpt>
    <trkpt lat="18.70912374" lon="81.910785026">
    </trkpt>
    <trkpt lat="18.700072554" lon="81.907843458">
    </trkpt>
    <trkpt lat="18.692462913" lon="81.907657717">
    </trkpt>
    <trkpt lat="18.67736592" lon="81.903163216">
    </trkpt>
    <trkpt lat="18.661242568" lon="81.900639647">
    </trkpt>
  </trkseg>
</trk>
</gpx>
```
</details>

It is generated from a .geojson polygon shape in python using this code:  
<details>

```
import geopandas as gpd
from shapely.geometry import shape

# geo = a geojson containing a polygon loaded as a dict
BLOCK_ID = '1445'
bdf1 = gpd.GeoDataFrame({'geometry':[shape(geo).simplify(0.001)]}, crs="EPSG:4326")
bdf1.boundary.to_file(os.path.join(gpxFolder,f"{BLOCK_ID}.gpx"), 'GPX')
```
</details>

### Which iD Editor versions do you see the issue on?

Released version at openstreetmap.org/edit

### Which browsers are you seeing this problem on?

Firefox, Chrome

### PR DESCRIPTION
**Pull request by answerquest**
update params structure for the iD editor hosted at openstreetmap.org/edit : have to keep the params after a # instead of ?

Solves https://github.com/openstreetmap/iD/issues/9093  and https://github.com/openstreetmap/openstreetmap-website/issues/3540

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -67,7 +67,8 @@ of iD (e.g. `https://ideditor-release.netlify.app`), the following parameters ar
 
 When constructing a URL to an instance of iD embedded in the OpenStreetMap Rails
 Port (e.g. `http://www.openstreetmap.org/edit?editor=id`), the following parameters
-are available as **regular URL query parameters**:
+are available as **URL hash-query parameters, to put after a # instead of a ?**.  
+_Example:_ `https://www.openstreetmap.org/edit#gpx=https://tasks.hotosm.org/project/592/task/16.gpx`
```
Comment (mmd-osm) [inline API.md]: Unfortunately, the link to that GPX file doesn't seem to work here.

Comment (answerquest) [reply API.md]: Yes, but same link is mentioned in the same page above also. I assumed keeping the same here will give a consistent logical flow to help someone reading the documentation to follow properly. I do have valid GPX links but it'll end up exposing my server to more bots etc. Let me try hosting one in a safe place.

Comment (answerquest) [reply API.md]: ok how about this: https://www.openstreetmap.org/edit#gpx=https://gist.githubusercontent.com/answerquest/9445352b60ca5b44714675eae00f243a/raw/56a6343a29223318f4a697bfd16cbb2c3b8155ad/sample_boundary.gpx

Comment (answerquest) [reply API.md]: updated. pls tell if you need me to change the gpx link in top portion of the page too in the standalone iD section. I'll prefer to leave that be.

Comment (mmd-osm) [reply API.md]: Example works fine now, thanks!

**Comment by tyrasd** [comment]
I added some clarifications to this PR in 93713a41adb745ab2b25309d752258eba1d6057c: In fact, some of the listed parameters are actually query parameters (e.g. `lat`,`lon`,`zoom`,`node`,`way`,`relation`, `locale`).

Slightly related: There exist a small bug in the openstreetmap website which in some situations caused "all" query parameters to be interpreted like hash parameters. https://github.com/openstreetmap/openstreetmap-website/pull/3571 has a fix for that.

**Pull request merged**